app/src/main/java/at/roteskreuz/stopcorona/model/entities/configuration/DbConfiguration.kt

```diff
@@ -30,7 +30,16 @@ data class DbConfiguration(
     /**
      * Time of updating.
      */
-    val cacheTime: ZonedDateTime = ZonedDateTime.now()
+    val cacheTime: ZonedDateTime = ZonedDateTime.now(),
+
+    val exposureConfigurationMinimumRiskScore: Int = 1,                                                    // 1,
+    val exposureConfigurationDailyRiskThreshold: Int = 30,                                                 // 30,
```

Comment:
You are hardcoding default values inside DB entity definition. Please move values to the `Constants` if they should be as default value in case of null.
Btw. it should be default in Api entity.

Reply:
Merge code from https://github.com/austrianredcross/stopp-corona-android/pull/124

Reply:
@falkorichter